onnxruntime/core/providers/webnn/builders/impl/normalization_op_builder.cc

```diff
@@ -86,12 +110,42 @@ Status NormalizationOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder
   NodeAttrHelper helper(node);
   options.set("epsilon", helper.Get("epsilon", 1e-05f));
 
-  int64_t axis = helper.Get("axis", -1);
-  axis = HandleNegativeAxis(axis, rank);
-  std::vector<int32_t> axes{static_cast<int32_t>(axis)};
-  options.set("axes", emscripten::val::array(axes));
-
-  emscripten::val output = model_builder.GetBuilder().call<emscripten::val>("meanVarianceNormalization", input, options);
+  emscripten::val output = emscripten::val::undefined();
+  if (op_type == "LayerNormalization") {
+    int64_t axis = helper.Get("axis", -1);
+    axis = HandleNegativeAxis(axis, rank);
+    std::vector<int32_t> axes{static_cast<int32_t>(axis)};
+    options.set("axes", emscripten::val::array(axes));
+    output = model_builder.GetBuilder().call<emscripten::val>("meanVarianceNormalization", input, options);
+  } else if (op_type == "InstanceNormalization") {
+    std::vector<int32_t> axes;
+    for (size_t i = 2; i < rank; i++) {
+      axes.emplace_back(i);
+    }
+    options.set("axes", emscripten::val::array(axes));
+    output = model_builder.GetBuilder().call<emscripten::val>("meanVarianceNormalization", input, options);
+  } else if (op_type == "GroupNormalization") {
+    ORT_RETURN_IF_NOT(helper.HasAttr("num_groups"), "group must be provided.");
```

Comment:
```suggestion
    ORT_RETURN_IF_NOT(helper.HasAttr("num_groups"), "GroupNormalization num_group must be provided.");
```

Reply:
Would use actual ONNX attribute name in the comment to be clearer. Rest looks good.